drivers/char/tpm/tpm_ftpm_optee.c

```diff
@@ -76,37 +81,35 @@ static int ftpm_optee_tpm_op_send(struct tpm_chip *chip, u8 *buf, size_t len)
 		{ .attr = TEE_IOCTL_PARAM_ATTR_TYPE_NONE }
 	};
 	struct tee_ioctl_invoke_arg submit_command_args;
-	struct tee_shm *shm = NULL;
 
-	mutex_lock(&tpm_tee_context.response_buffer_lock);
+	if (tpm_tee_context.response_length != 0) {
+		rc = -EIO;
+		goto cleanup;
+	}
+
+	if (len > FTPM_SHM_SIZE - FTPM_RESPONSE_SIZE_MAX) {
```

Comment:
Suggest to parenthesize as: `len > (FTPM_SHM_SIZE - FTPM_RESPONSE_SIZE_MAX)`

Reply:
Sounds good.